src/aml_service/index.js

```diff
@@ -0,0 +1,86 @@
+'use strict'
+
+const yotiRequest = require('../yoti_request');
+const AmlResultClass = require('../yoti_request/aml.result').AmlResult;
+const Payload = require('../yoti_request/payload').Payload;
+
+const AmlResultObj = function(rawResult){
+  this.onPepList = rawResult['on_pep_list'];
+  this.onFraudList = rawResult['on_fraud_list'];
+  this.onWatchList = rawResult['on_watch_list'];
+}
+
+AmlResultObj.prototype = {
+  /**
+   * Check if user is a politically exposed person.
+   *
+   * @returns {exports.AmlResult.isOnPepList}
+   */
+  isOnPepList() {
```

Comment:
Could use a getter if this AmlResultObj was a class, same for the field below. Would avoid the function call `amlResult.isOnPepList()` and become `amlResult.isOnPepList`

Reply:
I used a function here to be consistent with the other SDKs (PHP, Java) as this is the interface that clients will be using. But I am happy to use an attribute instead.

Reply:
Also `Country` doesn't need to be a type, I used a type here in case we need to add more `Country` attribute(s) in the future like `country_name`. But I am happy to squeeze this into `AmlAddress` as an attribute `countryCode`.